tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": [
+    "@tsconfig/node16/tsconfig",
+    "@tsconfig/strictest/tsconfig",
+    "@tsconfig/esm/tsconfig"
+  ],
+  "compilerOptions": {
+    "declaration": false,
+    "moduleResolution": "NodeNext",
+    "outDir": "dist-for-testing"
+  },
+  "include": ["bin", "src", "tests"],
+  "exclude": ["tests/fixtures"]
```

Comment:
Consider the possibility of moving `tests/fixtures` to the project root as well.